.gitignore

```diff
@@ -22,3 +22,5 @@ api/src/lib/generateGraphiQLHeader.*
 !.yarn/versions
 
 postgres
+
+docker-compose.override.yml
```

Comment:
@Josh-Walker-GM is this an artifact or does this file get generated by docker compose or something like that?

Reply:
@rainkinz Could you help answer this one? I think this was merged in from your PR to this one.

Reply:
`docker-compose.override.yml` is automatically merged with your `docker-compose.yml` when you do anything with docker-compose. You generally would put mounts to your local file system etc that are specific to you. For example I usually mount my `~/.aws` directory and history so that in the console it's all there. I can demo for you both if you want.

Reply:
Sorry, `docker-compose.override.yml` is manually created.